### PR TITLE
modified list-of-pairs to copy datasets as hidden

### PR DESCRIPTION
**Pull request by assuntad23**
Fix for papercut issue #10229 

It only fixes new collections that are created using the updated collection creator. 

Thanks to much help from @jmchilton , @dannon , and @mvdbeek !

### CODE REVIEW

**Review by mvdbeek — APPROVED** [review]
Nice!

File: lib/galaxy/model/__init__.py [hunk 1]
```diff
@@ -4593,6 +4593,7 @@ def copy_to_collection(self, collection, destination=None, element_destination=N
                 )
             else:
                 new_element_object = element_object.copy(flush=flush)
+                new_element_object.visible = False
```
Comment (mvdbeek) [inline lib/galaxy/model/__init__.py:4596]: @assuntad23 this is the complementary fix for existing collections. This sets all member HDAs of a newly copied collection to hidden. I can't think of a situation where they should ever remain visible.

**Comment by mvdbeek** [comment]
@galaxybot test this

File: lib/galaxy/model/__init__.py (hunk 1 above)
Comment (assuntad23) [reply lib/galaxy/model/__init__.py:4596]: Thanks Marius!

**Review by assuntad23 — COMMENTED** [review]
I think this is good to go, but we may need someone else to approve?

**Comment by mvdbeek** [comment]
This is fine, thanks @assuntad23!

**Pull request merged**